setup.py

```diff
@@ -68,6 +68,8 @@ def find_package_data(package, data_dir):
     install_requires=[
         'ansible >= 2.5; python_version < "3.8"',
         'ansible-core; python_version >= "3.8"',
+        'importlib_resources; python_version < "3.7"',
+        'importlib_metadata; python_version < "3.8"',
```

Comment:
given on EL8 we use py3.11 now, this is safe to merge once we turn of the el7 builders.

Reply:
Should we then refactor this in a "drop Python < 3.8 compatibility" PR?

Reply:
I think downstream it's still py3.6 (with whatever ansible is compatible with that)

Reply:
Do we still need to care about this or can we assume Python 3.8+ by now?